spec/models/manageiq/providers/ansible_tower/automation_manager/job_spec.rb

```diff
@@ -1,3 +1,36 @@
 describe ManageIQ::Providers::AnsibleTower::AutomationManager::Job do
+  let(:job) { FactoryGirl.create(:ansible_tower_job) }
+
   it_behaves_like 'ansible job'
+
+  describe '#raw_stdout_via_worker' do
```

Comment:
can you move new spec code inside `it_hehaves_like 'ansible job'`?

Reply:
It won't work since this feature is also used in Embedded Ansible and in that case the task requires `embedded_ansible` role to be provided by the server. Embedded Ansible has its implementation adjusted in this PR https://github.com/ManageIQ/manageiq/pull/17712 and their very own specs are available [here](https://github.com/ManageIQ/manageiq/blob/master/spec/models/manageiq/providers/embedded_ansible/automation_manager/job_spec.rb)

In other words: If moved to the shared example these specs would fail for Embedded Ansible.

Reply:
@tumido it's likely doable to share the spec.  Try move [this line](https://github.com/ManageIQ/manageiq/blob/master/spec/models/manageiq/providers/embedded_ansible/automation_manager/job_spec.rb#L15) to L3 and the shared specs will receive it.

Reply:
@jameswnl :+1: I've moved it. Let me know if the current spec layout is acceptable. :wink: I needed to change the context flow for Embedded Ansible job_spec. Please review the core PR as well. https://github.com/ManageIQ/manageiq/pull/17712